sgv2-docsapi/src/main/java/io/stargate/sgv2/docsapi/service/schema/JsonSchemaManager.java

```diff
@@ -70,36 +69,37 @@ public Uni<JsonNode> getJsonSchema(Uni<Schema.CqlTable> table) {
   /**
    * Assigns a JSON schema to a table.
    *
-   * @param keyspace the keyspace of the table
-   * @param table the table
+   * @param namespace the namespace of the collection
+   * @param collection the collection to attach JSON schema to
    * @param schema the JSON schema to assign
    * @return a Uni with a success boolean
    */
-  public Uni<QueryOuterClass.Response> attachJsonSchema(
-      String keyspace, String table, JsonNode schema) {
-    return Uni.createFrom()
-        .item(() -> jsonSchemaFactory.getSyntaxValidator().validateSchema(schema))
-        .flatMap(
-            report -> {
-              if (report.isSuccess()) {
-                ObjectNode wrappedSchema = objectMapper.createObjectNode();
-                wrappedSchema.set("schema", schema);
-                StargateBridge bridge = requestInfo.getStargateBridge();
-                return bridge.executeQuery(
+  public Uni<JsonNode> attachJsonSchema(
+      String namespace, Uni<Schema.CqlTable> collection, JsonNode schema) {
```

Comment:
I left `namespace` as a parameter here, because I couldn't figure out an obvious way to get the keyspace from a Schema.CqlTable object